grass7/imagery/i.landsat8.swlst/column_water_vapor.py

```diff
@@ -371,16 +371,16 @@ def _big_cwv_expression(self):
             self._denominator_for_ratio_big(mean_ti=string_for_mean_ti)
 
         cwv = ('eval('
-               '\ \n  ti_mean = {tim},'
-               '\ \n'
-               '\ \n  tj_mean = {tjm},'
-               '\ \n'
-               '\ \n  numerator = {numerator},'
-               '\ \n'
-               '\ \n  denominator = {denominator},'
-               '\ \n'
-               '\ \n  rji = numerator / denominator,'
-               '\ \n'
+               r'\ \n  ti_mean = {tim},'
```

Comment:
Does the expression still work with the change to raw string?

Reply:
Does @NikosAlexandris have an idea here?

Reply:
Shall I revert this part? I really want to merge this PR.

Reply:
I just ran a little test. Conversion to raw strings seem to break the mapcalculator expressions:

```
import grass.script as gscript

expression = 'mc_test=1*2+\ \n3' 
expression_r = r'mc_testr=1*2+\ \n3' 

# This works
gscript.mapcalc(expression)
```

However, this:
```
gscript.mapcalc(expression_r)

```
throws the following error message:
```
Invalid map <\>
Parse error
ERROR: parse error
ERROR: An error occurred while running r.mapcalc with expression:
       mc_testr=1*2+\ \n3
ERROR:root:Internal Python error in the inspect module.
Below is the traceback from this internal error.
```
(...)

So, yes, I am afraid the part where strings are converted to raw strings should be reverted...

Reply:
> Shall I revert this part? I really want to merge this PR.

I didn't the time to look if raw strings would make any difference.
I can check upon this tonight.

Reply:
@NikosAlexandris did you have a chance to check?

Reply:
Reverted in 1d1594392ea1afadf1d1a9082c5faa70c6637ab6

Thanks for having spotted this issue.